lib/install/proto/proto.go

```diff
@@ -69,7 +69,12 @@ func KeyToProto(key ops.SiteOperationKey) *OperationKey {
 
 // HasSpecificPhase determines if this request is for a specific phase (other than root)
 func (r *ExecuteRequest) HasSpecificPhase() bool {
-	return r.Phase != nil && !r.Phase.IsResume()
+	return !r.HasResume()
+}
+
+// HasResume determines if this is a request to resume an operation
+func (r *ExecuteRequest) HasResume() bool {
+	return r.Phase == nil || r.Phase.IsResume()
```

Comment:
Actually, you're both correct - I double-checked and the resume operation always uses the root phase ID so this will be changed to match the expectation.